README.md

```diff
@@ -1,39 +1,90 @@
 
 ## Motivation
 
-This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits.
+This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits. 
+
+
+## Examples
+
+(The following examples also appear in the README.md of Babel transform plugin for this proposal.)
+
+
+### Decimal Literals
+
+```js
+let budget = 1_000_000_000_000;
+
+// What is the value of `budget`? It's 1 trillion!
+// 
+// Let's confirm:
+console.log(budget === 10 ** 12); // true
+```
+
+### Binary Literals
+
+```js
+let nibbles = 0b1010_0001_1000_0101;
+
+// Is bit 7 on? It sure is!
+// 0b1010_0001_1000_0101
+//             ^
+//             
+// We can double check: 
+console.log(!!(nibbles & (1 << 7))); // true
+```
 
-For example:
+### Hex Literal
 
 ```js
-var thousands = 10_000; // Instead of 10000.
-var credit_card_number = 1234_5678_9012_3456; // Instead of 123456789012345.
-var social_security_number = 999_99_9999; // Instead of 999999999.
-var pi = 3.14_15; // Instead of 3.1415
-var bytes = 0b11010010_01101001_10010100_10010010; // Instead of 0b11010010011010011001010010010010.
-var 0xCAFE_F00D; // Instead of 0XCAFEF00D.
+// Messages are sent as 24 bit values, but should be 
+// treated as 3 distinct bytes:
+let message = 0xA0_B0_C0;
+
+// What's the value of the upper most byte? It's A0, or 160.
+// We can confirm that:
+let a = (message >> 16) & 0xFF; 
+console.log(a.toString(16), a); // a0, 160
+
+// What's the value of the middle byte? It's B0, or 176.
+// Let's just make sure...
+let b = (message >> 8) & 0xFF;
+console.log(b.toString(16), b); // b0, 176
+
+// What's the value of the lower most byte? It's C0, or 192.
+// Again, let's prove that:
+let c = message & 0xFF;
+console.log(c.toString(16), b); // c0, 192
 ```
 
-## Strawman
+### Octal Literal
+
+*hand wave emoji*
+
+Octals are great for permissions, but also look better when represented in `0o0000` form. No real benefit with separators here. 
+
+
+## Specification 
 
 ### Semantics
 
 This feature is designed to have no impact on the interpretation semantics of numeric literals: `_` are to be ignored by interpreters and should have no effect. They are meant **exclusively** as a visual clue to aid development and have no runtime semantics.
 
 ### Syntax
 
-We want to optimize to cover the most common use cases and discourage patterns that would be frowned upon in style guides later on.
 
-With that in mind, here is what we think is a good balance:
+The following grammar represents the Stage 1 criteria, which is: 
 
-- to use the `_` character.
-- only one consecutive underscore is allowed.
-- only between digits (not allowed at the beginning or end of literals).
+1. No leading or trailing separator.
+2. No multiple adjacent separator.
```

Comment:
Aren't these still equivalent? That is, isn't (a) "No leading or trailing separator." equivalent to "only between digits" (i.e. if it is only between digits, doesn't that imply that leading and trailing are disallowed? and vice versa?), and (b) "No multiple adjacent separator" equivalent to "only one consecutive underscore is allowed"?